### PR TITLE
Add authorship component

### PR DESCRIPTION
**Pull request by hcorson-dosch-usgs**
This PR adds in and customizes the authorship component from Vue scratch. I've drafted the text of the attributions - feel free to edit.

### CODE REVIEW

**Comment by hcorson-dosch-usgs** [comment]
Updated contributions text to be more general

File: src/assets/text/authors.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    // currently only 'fullName', 'firstName', 'initials', 'profile_link', and 'contribution' are used
+    primaryAuthors: [
+      {
+        firstName: 'Cee',
+        lastName: 'Nell',
+        fullName: 'Cee Nell',
+        initials: 'CN',
+        profile_link: 'https://www.usgs.gov/staff-profiles/cee-nell',
+        role: 'developer',
+        contribution: 'contributed to pipeline development and creation of the prebuilt svg paths, developed the website and d3 animation, and reviewed code'
```
Comment (ceenell) [inline src/assets/text/authors.js:12]:
```suggestion
        contribution: 'contributed to the data processing pipeline, developed the website, and created the d3 animation'
```

File: src/assets/text/authors.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      },
+      {
+        firstName: 'Lindsay',
+        lastName: 'Platt',
+        fullName: 'Lindsay Platt',
+        initials: 'LP',
+        profile_link: 'https://www.usgs.gov/staff-profiles/lindsay-rc-platt',
+        role: 'developer',
+        contribution: 'developed the pipeline to pull historic and current groundwater levels, led the development of the prebuilt svg paths, and reviewed code'
```
Comment (ceenell) [inline src/assets/text/authors.js:21]:
```suggestion
        contribution: 'developed the data processing pipeline to pull historic and current groundwater levels, and generated the svg map'
```

**Review by ceenell — APPROVED** [review]
Made a few minor suggestions to simplify the contributions. Mainly, most people won't know what is meant by pre-building svgs, so I adapted to make it more general. Pipelines are a more familiar concept but I added "data processing pipeline" to make that clear. For your contribution, I'll leave it to you to define, but you could say that you're the perosn maintaining the site or something like that to acknowledge that you have done more than just regenerate for a new time period.

**Comment by hcorson-dosch-usgs** [comment]
Thanks! I'd made pretty much the identical edits that you did yesterday and then forgot to push them 🤦‍♀️. Your edits are in now, and I'll merge and rebuild beta then prod

**Pull request merged**